draft-ietf-teep-protocol.md

```diff
@@ -334,6 +334,9 @@ query-request = [
     ? supported-freshness-mechanisms => [ + $freshness-mechanism ],
     ? challenge => bstr .size (8..512),
     ? versions => [ + version ],
+    ? attestation-payload-format => text,
+    ? attestation-payload => bstr,
+    ? suit-reports => [ + bstr ],
```

Comment:
The body of the document needs to be updated as well with field descriptions.
Also add an err-code value such as
ERR_ATTESTATION_REQUIRED = 7

Reply:
@dthaler 
Add the descriptions of paramaters requied for attesting the TAM.
Also add it for ERR_ATTESTATION_REQUIRED.